extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/logs/OpenTelemetryLogHandler.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.opentelemetry.runtime.logs;
+
+import static io.quarkus.opentelemetry.runtime.config.build.OTelBuildConfig.INSTRUMENTATION_NAME;
+
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.logs.Logger;
+import io.opentelemetry.api.logs.Severity;
+
+public class OpenTelemetryLogHandler extends Handler {
+
+    @Override
+    public void publish(LogRecord record) {
+        Logger openTelemetry = GlobalOpenTelemetry.get().getLogsBridge().get(INSTRUMENTATION_NAME);
```

Comment:
We need to make sure there is no other way to get the logger... This is likely to be a source of startup order problems.
Ideally the logger must come from CDI.

Reply:
One thing we can do is to cache log entries until OTel is ready and replay the entries...
Actually we do exactly that on the standard logging, because it must be present before quarkus is started...
I'm concerned with memory usage, but we could have a cap in the amount of stored bytes and discard after that.

Reply:
The appender will be called by our login backend, so if it's already done there, we don't need to do it here.
I think the real issue is how to access the GlobalTelemetry object, if there is no better way that what I didn't: at runtime when the appended is called and not at deployment RUN_TIME when the appended is constructed, then I'll create an holder class.

Reply:
I swtich to using Arc to retrieve the OpenTelemetry object from the CDI context.
Not sure if it would be costly or not, I may need to retrieve it once and cache it. At least all tests passed with that change which was not the case before.